### PR TITLE
feat: add i18 setup

### PR DESCRIPTION
**Pull request by AAKlimkov**

### CODE REVIEW

File: src/i18n.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { initReactI18next } from "react-i18next";
+import LanguageDetector from "i18next-browser-languagedetector";
+import Backend from "i18next-http-backend";
+
+void i18n
+    .use(initReactI18next)
+    .use(LanguageDetector)
+    .use(Backend)
+    .init({
+        supportedLngs: ["en", "ru"],
```
Comment (shev7) [inline src/i18n.ts]: Create enum for langs

**Review by shev7 — CHANGES_REQUESTED** [review]

File: src/i18n.ts (hunk 1 above)
Comment (AAKlimkov) [reply src/i18n.ts]: fixed

File: public/locales/en/translation.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "Login": "Login",
+  "Signup": "Signup",
+  "Welcome Back": "Welcome Back",
+  "Register now": "Hello world"
```
Comment (savelievamareena) [inline public/locales/en/translation.json]: Shouldn't it be "Register now" too?

File: src/i18n.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+import LanguageDetector from "i18next-browser-languagedetector";
+import Backend from "i18next-http-backend";
+
+export enum Languages {
+    ru = "ru",
+    en = "en",
```
Comment (shev7) [inline src/i18n.ts]: enum keys should be capitalized
```suggestion
    Ru = "ru",
    En = "en",
```

Comment (AAKlimkov) [reply src/i18n.ts]: Fixed

File: public/locales/en/translation.json (hunk 2 above)
Comment (AAKlimkov) [reply public/locales/en/translation.json]: fixed

**Review by RC-3222 — APPROVED** [review]

File: src/i18n.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+import LanguageDetector from "i18next-browser-languagedetector";
+import Backend from "i18next-http-backend";
+
+export enum Languages {
+    Ru = "ru",
+    En = "en",
+}
```
Comment (savelievamareena) [inline src/i18n.ts:9]: I propose to move language constants to the global constants folder so they become easily and logically accessible for different modules/components (we also need them in the Header for the dropdown language selector)

Comment (AAKlimkov) [reply src/i18n.ts:9]: i add plugin to detect browser language. I think it should resolve this problem

**Review by shev7 — APPROVED** [review]

**Pull request merged**